src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Color;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.RectangleF;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_AMOUNT = 5;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final int MAX_RANDOM_NUMBER = 50;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public final Figure getRandomFigure() {
+        switch (random.nextInt(FIGURE_AMOUNT)) {
```

Comment:
Lets create 5 private methods getRandomCircle(), getRandomSquare(), ... and call them in switch-case construction, this will increase code readability. (Remember private methods should follow after public as helper ones)